lib/child_process.js

```diff
@@ -297,12 +297,14 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
-var _deprecatedCustomFds = internalUtil.deprecate(function(options) {
-  options.stdio = options.customFds.map(function(fd) {
-    return fd === -1 ? 'pipe' : fd;
-  });
-}, 'child_process: options.customFds option is deprecated. ' +
-   'Use options.stdio instead.');
+const _deprecatedCustomFds = internalUtil.deprecate(
+  function deprecateCustomFds(options) {
+    options.stdio = options.customFds.map(function(fd) {
```

Comment:
when removing the return statement i get the no-confusing-arrow lint rule being thrown complaining about using arrow functions ambigiously with conditional expressions.